scripts/new_upstream_snapshot.py

```diff
@@ -339,20 +339,24 @@ def refresh_patches(commitish) -> bool:
     if rc not in [0, 2]:  # 2 means there were no quilt patches to pop
         raise CliError(f"'quilt pop -a' unexpectedly returned {rc}.")
 
-    # Now commit the refreshed patches
+    # Now commit the refreshed patches and add to changelog
     patches = capture(
         "git diff --name-only debian/patches/", check=False
     ).stdout.splitlines()
-    if patches:
-        commit_msg = (
-            f"refresh patches against {commitish}\n\n"
-            f"patches: \n" + "\n".join(patches)
-        )
-        sh(f"git commit --no-verify -m '{commit_msg}' {' '.join(patches)}")
-        return True
-    else:
+    if not patches:
         print("No patches needed refresh")
         return False
+    
+    commit_msg = (
+        f"refresh patches against {commitish}\n\n"
+        f"patches: \n" + "\n".join(patches)
+    )
+    sh(f"git commit --no-verify -m '{commit_msg}' {' '.join(patches)}")
+    patch_texts = [p.replace("debian/patches/", "d/p/") for p in patches]
```

Comment:
Yes funnily enough this was my first thought no this character for character before switching to an alternative proposal. +1